### ISSUE TITLE
Checkpoints not found

### ISSUE THREAD

**Issue by naitian** [maxrmorrison/pyfoal#21]
From the documentation, I use `pyfoal.DEFAULT_CHECKPOINT`, but it looks like this path (`assets/checkpoints`) doesn't exist. Are we supposed to train the model ourselves? Are there pre-trained weights available for download somewhere?

Thanks!

**Comment by maxrmorrison** [comment]
I do not currently have pre-trained weights. My concern with releasing pre-trained weights is—while the rough idea of RADTTS is provided in this repo—it is not the official version, and contains some differences. Until those are mended, I've been keeping the bar for using that a bit higher. Also, P2FA and my improvements over my baseline RADTTS (in config/) both outperform each other on different metrics included in pyfoal. So it's not yet clear that RADTTS is strictly improving over much older methods.

TLDR; training works, P2FA works, but no RADTTS checkpoint until I've verified that implementation is correct relative to the original paper.

**Comment by stepfunction83** [comment]
It would be helpful if RADTTS were not the default option to avoid confusion.
